src/promptflow-tools/promptflow/tools/open_source_llm.py

```diff
@@ -0,0 +1,466 @@
+import functools
+import json
```

Comment:
Microsoft header is required for the file.

Reply:
I'm not sure this is standard in this repo - on a spot check I don't see any other files with the banner?